README.md

```diff
@@ -4,6 +4,7 @@
 [![Downloads](https://img.shields.io/npm/dm/tslint-microsoft-contrib.svg)](https://npmjs.org/package/tslint-microsoft-contrib)
 [![Build Status](https://travis-ci.org/Microsoft/tslint-microsoft-contrib.svg?branch=master)](https://travis-ci.org/Microsoft/tslint-microsoft-contrib)
 [![Join the chat at https://gitter.im/Microsoft/tslint-microsoft-contrib](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/Microsoft/tslint-microsoft-contrib?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Build Status](https://dev.azure.com/tslint-microsoft-contrib/tslint-microsoft-contrib/_apis/build/status/tslint-microsoft-contrib)](https://dev.azure.com/tslint-microsoft-contrib/tslint-microsoft-contrib/_build/latest?definitionId=1)
```

Comment:
> `[Build Status]`

Could you change the image alt texts to read `[TravisCI Build Status]` and `[Azure Pipelines Build Status]`? It's a bit confusing to have two of the same.